### DISCUSSION TITLE
풀리퀘 확인에 대해 질문드립니다.

### DISCUSSION THREAD

**Discussion by solidcellaMoon** [devrel-kr/HackaLearn#336] [1 Q&A - HackaLearn 운영] [answered]
안녕하세요! 챌린지 PR 확인 건에 대해 질문 드립니다.
저희 팀은 충돌을 피하기 위해 팀원 한명씩 순서대로 PR을 올리고 있는데,
아직 저희 팀이 올린 PR이 확인되지 않아 시간이 지체되고 있는 상황입니다.
가능하다면 #317 번, #295 번 PR을 확인해주시면 감사하겠습니다.

아래는 PR에 대한 질문입니다.
1. 충돌이 나더라도 팀원 여러명이 PR을 동시에 제출해도 되는지 궁금합니다.
2. 다른 팀원이 대신 PR을 올리는 것도 가능한지 궁금합니다.
(예: A팀원이 부재중인 B팀원 대신 B팀원의 스크린샷 주소를 PR)

**Comment by justinyoo** [comment, marked as answer]
@solidcellaMoon 우선은 남아있는 PR을 먼저 올려 주세요 오늘 밤 자정 전 까지 생성한 PR에 한해서는 16일 낮 12시까지는 수정하는 것에 한해 허용합니다. PR 생성 후 다른 PR을 처리하는 과정에서 생기는 컨플릭은 하나씩 해결해 나가는 것으로 하죠.

**Reply by justinyoo** [reply]
참고로 다른 팀원 것을 대신 올려주는 것은 해당 팀원의 아이디로 로그인을 하지 않는 이상 대신 올려주는 것은 시스템상 불가능합니다.

**Reply by solidcellaMoon** [reply]
넵 자세한 답변 감사드립니다~!
